modules/ROOT/pages/6.3-release-notes.adoc

```diff
@@ -0,0 +1,697 @@
+
+For {productname} 6.3, a fix was applied to the core `SelectionOverrides` settings to ensure the user selection restores from the ‘bookmark’ positioning before and after an inline element with a `contenteditable="false"` attribute is created.
+
+Hitting the *Return* (or *Enter*) after creating a merge tag now moves the insertion point to a new line, as expected.
+
+
+=== Inline mode did not respect the `content_css_cors` option.
+
+In previous versions of {productname}, the `tinymce.styleSheetLoader` global was not configured by the `content_css_cors` option, when specified.
+
+Consequently, stylesheets loaded by {productname} onto the outer document page did not have the `crossOrigin="anonymous"` attribute. This particularly effected {productname} instances running in inline mode, because the `tinymce.styleSheetLoader` global is used to load stylesheets used by inline editors. (Stylesheets for {productname} editors running in iframe mode are specific to the editor.)
+
+Without the `crossOrigins="anonymous"` attribute being set, no external stylesheets were loaded. And inline editors with the `pageembed` plugin loaded and the `tiny_pageembed_inline_styles` option set to `true` would, as a result, return an error when attempting to insert any iframse through the dialog.
+
+As of {productname} 6.3, the `content_css_cors` option now configures the `tinymce.dom.styleSheetLoader` global when specified. This allows the `crossOrigin="anonymous"` attribute to be added to all stylesheets loaded by the editor — including editors running in inline mode — when the attribute is set to true.
+
+=== The insertion point moved to the previous line when a text pattern executed a `mceInsertContent` command on Firefox
+
+In previous versions of {productname}, when running on Firefox and after pressing Enter or Return to execute an `mceInsertContent` command (to, for example, add a merge tag to a {productname} document), xref:content-behavior-options.adoc#text_patterns[text_patterns] moved the insertion point to the previous line.
+
+This error has been corrected in {productname} 6.3. When running {productname} 6.3 on Firefox and creating a merge tag using `mceInsertContent`, pressing the Enter or Return to generate the desired merge tag moves the insertion point to the end of said newly-created merge tag, as expected.
+
+
+[[security-fixes]]
+== Security fixes
+
+{productname} 6.3 includes fixes for the following security issues:
+
+=== HTML in messages for the `WindowManager.alert` and `WindowManager.confirm` APIs was not properly sanitized.
+
+Some calls to `DomFactory.fromHtml` were escaping the text being substituted into the HTML `string`, resulting in an isolated alert being displayed within the {productname} editor viewport. 
+
+{productname} 6.3 introduced `dompurify` into the core dependencies to reduce the risks associated with XSS attacks.
```

Comment:
5.10.7 adds dompurify. For the 6.x it's been in there since the feist version. So maybe just strike this from the notes here.

Reply:
As per this [commit](https://github.com/tinymce/tinymce-docs/commit/26c4e66d187d8a08e2a5741ed37947162cc07ef1) and its message:

> Edits to ‘HTML in messages for the `WindowManager.alert` and `WindowManager.confirm` APIs was not properly sanitized’ section as per @spocke’s notes.